AdSecGH/Parameters/AdSecRebarGroupGoo.cs

```diff
@@ -188,42 +95,39 @@ public override string ToString() {
               if (stress.Stress.Value != 0) {
                 IQuantity quantityStress = new Pressure(0, DefaultUnits.StressUnitResult);
                 string unitstressAbbreviation = string.Concat(quantityStress.ToString().Where(char.IsLetter));
-                m_preLoad = $", {Math.Round(stress.Stress.As(DefaultUnits.StressUnitResult), 4)}{unitstressAbbreviation} prestress";
+                m_preLoad
+                  = $", {Math.Round(stress.Stress.As(DefaultUnits.StressUnitResult), 4)}{unitstressAbbreviation} prestress";
               }
             } catch (Exception) {
               var strain = (IPreStrain)longitudinal.Preload;
               if (strain.Strain.Value != 0) {
                 string unitstrainAbbreviation = Strain.GetAbbreviation(DefaultUnits.MaterialStrainUnit);
-                m_preLoad = $", {Math.Round(strain.Strain.As(DefaultUnits.MaterialStrainUnit), 4)}{unitstrainAbbreviation} prestress";
+                m_preLoad
+                  = $", {Math.Round(strain.Strain.As(DefaultUnits.MaterialStrainUnit), 4)}{unitstrainAbbreviation} prestress";
               }
             }
           }
         }
 
         try {
-          var temp = (ITemplateGroup)Value.Group;
           m_ToString = $"Template Group, {Value.Cover.UniformCover.ToUnit(DefaultUnits.LengthUnitGeometry)} cover";
         } catch (Exception) {
           try {
-            var perimeter = (IPerimeterGroup)Value.Group;
             m_ToString = $"Perimeter Group, {Value.Cover.UniformCover.ToUnit(DefaultUnits.LengthUnitGeometry)} cover";
           } catch (Exception) {
             try {
-              var arc = (IArcGroup)Value.Group;
               m_ToString = "Arc Type Layout";
             } catch (Exception) {
               try {
-                var cir = (ICircleGroup)Value.Group;
                 m_ToString = "Circle Type Layout";
               } catch (Exception) {
                 try {
-                  var lin = (ILineGroup)Value.Group;
                   m_ToString = "Line Type Layout";
                 } catch (Exception) {
                   try {
-                    var sin = (ISingleBars)Value.Group;
                     m_ToString = "SingleBars Type Layout";
                   } catch (Exception) {
+                    /* don't expect to fail */
                   }
                 }
               }
```

Comment:
ok, why you remove those? I think the code is meant to throw an exception? I guess this is never happening because it the cast simply return `null` ? I think this needs to become `if ( Value.Group is IArcGroup arcGroup)`